app/controllers/content_items_controller.rb

```diff
@@ -56,4 +63,16 @@ def error_403(exception)
   def error_notfound
     render plain: 'Not found', status: :not_found
   end
+
+  def ab_testing_group
+    request.headers["HTTP_GOVUK_ABTEST_EDUCATIONNAVIGATION"] == "B" ? "B" : "A"
+  end
+
+  def new_navigation_enabled?
+    ENV['ENABLE_NEW_NAVIGATION'] == 'yes'
+  end
+
+  def content_is_tagged_to_a_taxon?
```

Comment:
It's a new document type that defines the new navigation structure. A taxonomy is a tree of taxons. Content is tagged to these taxon nodes essentially as leaf nodes.